mozilla-mobile/android-components/components/feature/downloads/src/main/res/values/strings.xml

```diff
@@ -2,22 +2,34 @@
 <resources>
     <!-- Name of the "notification channel" used for displaying download notification. See https://developer.android.com/training/notify-user/channels -->
     <string name="mozac_feature_downloads_notification_channel">Downloads</string>
-    <!-- Text shown on the first row of an ongoing download notification. -->
-    <string name="mozac_feature_downloads_ongoing_notification_title">Downloading</string>
-    <!-- Text shown on the second row of an ongoing download notification. -->
-    <string name="mozac_feature_downloads_ongoing_notification_text">Download in progress</string>
 
+    <!-- Text shown on the second row of a paused download notification. -->
+    <string name="mozac_feature_downloads_paused_notification_text">Download paused</string>
     <!-- Text shown on the second row of an completed download notification. The filename is shown on the first row. -->
-    <string name="mozac_feature_downloads_completed_notification_text">Download completed.</string>
+    <string name="mozac_feature_downloads_completed_notification_text2">Download completed</string>
     <!-- Text shown on the second row of an failed download notification. The filename is shown on the first row. -->
-    <string name="mozac_feature_downloads_failed_notification_text">Failed to download.</string>
+    <string name="mozac_feature_downloads_failed_notification_text2">Download failed</string>
 
     <!-- Alert dialog confirmation before download a file, this is the title. -->
-    <string name="mozac_feature_downloads_dialog_title">Download file</string>
+    <string name="mozac_feature_downloads_dialog_title2">Download (%1$s)</string>
```

Comment:
@sblatz: Could you please comment what the placeholder is for here since the string added one that wasn't there before?
thanks! :)